language/move-model/src/translator/exp_generator.rs

```diff
@@ -0,0 +1,279 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
Can you move this one level up? The idea of the translator sub-module was that it does not contain public functionality. exp_rewriter is for example a similar generic module, and it lives one level up.